hypertrace-graphql-labels-schema-api/src/main/java/org/hypertrace/graphql/label/schema/Label.java

```diff
@@ -4,7 +4,7 @@
 import org.hypertrace.core.graphql.common.schema.id.Identifiable;
 
 @GraphQLName(Label.TYPE_NAME)
-public interface Label extends Identifiable, LabelData, LabeledEntities {
+public interface Label extends Identifiable, LabelData, LabeledEntities, LabelApplicationRules {
```

Comment:
naming nit: interfaces should describe their behavior, a Label isn't a LabelApplicationRules, it contains them.

Reply:
Fixed